src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -97,15 +97,46 @@ public function getName()
 
     private function sanitizeLogs($logs)
     {
-        foreach ($logs as $i => $log) {
+        $errorContextById = array();
+        $sanitizedLogs = array();
+
+        foreach ($logs as $log) {
             $context = $this->sanitizeContext($log['context']);
-            if (isset($context['type'], $context['level']) && !($context['type'] & $context['level'])) {
-                $context['scream'] = true;
+
+            if (isset($context['type'], $context['file'], $context['line'], $context['level'])) {
```

Comment:
file and line are always set by the error handler so I'd say no need, do you think about something else?